packages/components/src/components/option/option.stories.ts

```diff
@@ -34,149 +35,151 @@ export const Default = {
 };
 
 /**
- * Use the `disabled` attribute to disable an option. Clicks will be suppressed until the disabled state is removed. Use the `checkbox` attribute to prefix a styled checkbox. Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ * Use the `variant` attribute to set the option’s variant.
+ * - `default`
+ * - `checkbox`: Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ *
  */
 
-export const DisabledCheckbox = {
-  name: 'Disabled x Checkbox',
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled',
-          values: [false, true]
-        },
-        y: {
-          type: 'attribute',
-          name: 'checkbox',
-          values: [false, true]
-        }
-      },
-      constants: { type: 'attribute', name: 'disabled', value: true },
-      args
-    });
-  }
+export const Variants = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <sd-select
+        class="h-[260px] w-[420px]"
```

Comment:
I think you should set the height in the wrapper div instead of the component itself

Reply:
Updated and removed widths